### PR TITLE
Ear 1294 front end for num curr unit dur

### PR DESCRIPTION
**Pull request by zimboflyman**
### What is the context of this PR?

Acceptance criteria
Number

GIVEN I'm on a question page
WHEN I've added a number answer
THEN I can add an "Or" option to the answer

Currency

GIVEN I'm on a question page
WHEN I've added a currency answer
THEN I can add an "Or" option to the answer

Unit

GIVEN I'm on a question page
WHEN I've added a unit answer
THEN I can add an "Or" option to the answer

Duration

GIVEN I'm on a question page
WHEN I've added a duration answer
THEN I can add an "Or" option to the answer

Textbox

GIVEN I'm on a question page
WHEN I've added a textbox answer
THEN I can add an "Or" option to the answer

Textarea

GIVEN I'm on a question page
WHEN I've added a textarea answer
THEN I can add an "Or" option to the answer

checkbox
dealt with in another ticket

Disable "Or" option toggle switch if there is more than one answer on a page

GIVEN I'm on a question page
WHEN there is more than one answer on the page
THEN the "Or" option toggle is disabled

### CODE REVIEW

File: eq-author/src/App/page/Design/QuestionPageEditor/AnswerTypeSelector/index.js [hunk 1]
```diff
@@ -75,13 +75,21 @@ class AnswerTypeSelector extends React.Component {
...
       message: QUESTION_ANSWER_NOT_SELECTED,
     });
     let hasDateRange = false;
     let hasOtherAnswerType = false;
+    
+    let mutuallyExclusive = false;
+    // TODO-----------------add this once options from backend is merged in------------------;;;
+    // const answers = Array.from(this.props.page.answers);
+    // mutuallyExclusive = { ...answers?.options?.some(({ mutuallyExclusive }) => mutuallyExclusive === true)};
+
```
Comment (MattElcock) [inline eq-author/src/App/page/Design/QuestionPageEditor/AnswerTypeSelector/index.js:92]: Does this need to be done now the backend is merged in?

**Review by MattElcock — CHANGES_REQUESTED** [review]

File: eq-author/src/App/page/Design/answers/BasicAnswer/index.js [hunk 2]
```diff
@@ -144,10 +144,10 @@ export const StatelessBasicAnswer = ({
           />
         </Field>
       )}
-      {type === "Percentage" && (
+      {type !== "Checkbox" && type !== "Radio" && (
         <ToggleWrapper data-test="toggle-wrapper" disabled={multipleAnswers}>
           <InlineField>
-            <Label>{`"Or" option`}</Label>
+          <Label for="toggle-or-option">{`"Or" option`}</Label>
```
Comment (MattElcock) [inline eq-author/src/App/page/Design/answers/BasicAnswer/index.js:150]:
```suggestion
          <Label htmlFor="toggle-or-option">{`"Or" option`}</Label>
```

**Review by MattElcock — CHANGES_REQUESTED** [review]

File: eq-author/src/App/page/Design/QuestionPageEditor/AnswerTypeSelector/index.js (hunk 1 above)
Comment (zimboflyman) [reply eq-author/src/App/page/Design/QuestionPageEditor/AnswerTypeSelector/index.js]: yes - thanks MAtt - nowupdated and test added

**Review by MattElcock — APPROVED** [review]
Amazing, nice one Shane.

**Review by jasehumphr — APPROVED** [review]

**Pull request merged**